RecoEgamma/EgammaPhotonProducers/src/ReducedEGProducer.cc

```diff
@@ -728,6 +638,13 @@ void ReducedEGProducer::produce(edm::Event& theEvent, const edm::EventSetup& the
     fillerPhotonPFClusterIso.fill();
     theEvent.put(std::move(photonPFClusterIsos[iid]),outPhotonPFClusterIsos_[iid]);
   }
+  //oot photon iso value maps
+  for (unsigned int iid=0; iid<ootPhotonPFClusterIsos.size(); ++iid) {
+    edm::ValueMap<float>::Filler fillerOOTPhotonPFClusterIso(*ootPhotonPFClusterIsos[iid]);
+    fillerOOTPhotonPFClusterIso.insert(outOOTPhotonHandle,ootPhotonPFClusterIsoVals[iid].begin(),ootPhotonPFClusterIsoVals[iid].end());
+    fillerOOTPhotonPFClusterIso.fill();
+    theEvent.put(std::move(ootPhotonPFClusterIsos[iid]),outOOTPhotonPFClusterIsos_[iid]);
+  }
```

Comment:
if you feel like it all the map filling can be shortened/simplified:
remove declaration of ootPhotonPFClusterIsos (it is needed only right when the filler fills the values and not in the full scope) and loop over ootPhotonPFClusterIsoVals (and all other cases)

```C++
   auto fillMap = [](auto refH, auto& vec, edm::Event& ev, const std::string& cAl = ""){
     typedef  edm::ValueMap<typename std::decay<decltype(vec)>::type::value_type> MapType;
     auto oMap = std::make_unique<MapType>();
     {
       typename MapType::Filler filler(*oMap);
       filler.insert(refH, vec.begin(), vec.end());
       filler.fill();
     }
     ev.put(std::move(oMap), cAl);
   };
//then the above becomes
int index = 0;
for (auto const& vals : ootPhotonPFClusterIsoVals){
 fillMap(outOOTPhotonHandle, vals, theEvent, outOOTPhotonPFClusterIsos_[index]);
 index++;
}
```